### PR TITLE
Prepare for refresh SDKs in bulk to release the extension mocking function - 4

### PR DESCRIPTION
**Pull request by ArthurMa1978**

### CODE REVIEW

File: sdk/iothub/Azure.ResourceManager.IotHub/CHANGELOG.md [hunk 1]
```diff
@@ -1,15 +1,15 @@
 # Release History
 
-## 1.2.0-beta.1 (Unreleased)
+## 1.1.1 (2023-11-29)
```
Comment (live1206) [inline sdk/iothub/Azure.ResourceManager.IotHub/CHANGELOG.md:3]: should this be `1.2.0`?

File: sdk/kusto/Azure.ResourceManager.Kusto/CHANGELOG.md [hunk 2]
```diff
@@ -1,15 +1,15 @@
 # Release History
 
-## 1.6.0-beta.1 (Unreleased)
+## 1.5.1 (2023-11-29)
```
Comment (live1206) [inline sdk/kusto/Azure.ResourceManager.Kusto/CHANGELOG.md:3]: should this be 1.6.0?

File: sdk/iothub/Azure.ResourceManager.IotHub/CHANGELOG.md (hunk 1 above)
Comment (ArthurMa1978) [reply sdk/iothub/Azure.ResourceManager.IotHub/CHANGELOG.md:3]: A minor version bump occurs only when the api-version is upgraded.

File: sdk/kusto/Azure.ResourceManager.Kusto/CHANGELOG.md (hunk 2 above)
Comment (ArthurMa1978) [reply sdk/kusto/Azure.ResourceManager.Kusto/CHANGELOG.md:3]: A minor version bump occurs only when the api-version is upgraded.

**Review by live1206 — APPROVED** [review]

**Pull request merged**